stat/distuv/distribution_test.go

```diff
@@ -158,8 +158,12 @@ func checkQuantileCDFSurvival(t *testing.T, i int, xs []float64, c cumulanter, t
 }
 
 func checkProbContinuous(t *testing.T, i int, x []float64, p probLogprober, tol float64) {
-	// Check that the PDF is consistent (integrates to 1).
-	q := quad.Fixed(p.Prob, math.Inf(-1), math.Inf(1), 1000000, nil, 0)
+	checkProbContinuousWithBounds(t, i, x, math.Inf(-1), math.Inf(1), p, tol)
+}
+
+func checkProbContinuousWithBounds(t *testing.T, i int, x []float64, lb float64, ub float64, p probLogprober, tol float64) {
+	// Check that the PDF is consistent (integrates to 1). Takes lower and upper bounds for X.
```

Comment:
```
// checkProbContinuousWithBounds check that the PDF is consistent,
// it integrates to 1 from the lower to upper bound.
func checkProbContinuousWithBounds(t *testing.T, i int, x []float64, lower, upper float64, p probLogprober, tol float64) {
```

and associated param changes in the body.

If there is no `checkProbContinuous`, remove the `WithBounds` suffix. This is testing code, so the API doesn't need to be friendly and efficient, just obviously conprehensible.

Reply:
There is :) I could add the bounds to the original function, but it'd cause a cascade of changes everywhere.

Reply:
OK, just the signature/doc change then.